data/campaigns/Eastern_Invasion/ART_MUSIC_LICENSE.txt

```diff
@@ -0,0 +1,406 @@
+//--------------------
+// MUSIC
+//--------------------
+cry from elensefar (pluft) - CC BY SA 4.0
+    https://forums.wesnoth.org/viewtopic.php?t=48565
+    https://soundcloud.com/pluft/cry-from-elensefar
+
+heavens (gabriel silver) - CC BY 4.0
+    remixed by dalas; I release my remix under CC BY 4.0
+    https://forums.wesnoth.org/viewtopic.php?t=48565
+    https://www.youtube.com/watch?v=AOvwWOGH7bQ&list=PLk-SdLbgd5ol62NX0q7ys5ZbmJFVAMzCF&index=2
+
+
+//--------------------
+// SOUNDS
+//--------------------
+wind (hackerb9) - CC 0
+    edited by dalas; I release the edited sound under CC BY 4.0
+    https://freesound.org/people/hackerb9/sounds/555246/
+
+gate-fall
+lich_die_wail
+magic-twilight
+magic-twilight-miss
+    made by dalas by modifying pre-existing wesnoth sounds
+    I release these under CC BY 4.0
+
+warhammer (Iwiploppenisse) - CC BY 4.0
+    https://freesound.org/people/Iwiploppenisse/sounds/156031/
+    modified by dalas by adding pre-existing wesnoth sounds
+    I release my modifications under CC BY 4.0
+
+
+
+
+
+################################################################################################################################################################################
+# ASSORTED IMAGES
+################################################################################################################################################################################
+//--------------------
+// ATTACKS
+//--------------------
+blast-wave
+sword-astral-short
+sword-orcish-flaming
+    made by dalas by modifying pre-existing wesnoth images
+    I release these under CC BY 4.0
+
+sword-orcish-swing
+    from the Reign of Lords add-on
+
+
+//--------------------
+// HALO
+//--------------------
+terror
+    created by dalas
+    I release these images under CC BY 4.0
+
+idle-flash
+fire-aura-small
+light-shield
+obscures-aura
+    made by dalas by modifying pre-existing wesnoth images
+    I release these under CC BY 4.0
+
+shadow-mage-halo
+    copied from the pre-existing wesnoth images in liberty
+
+
+//--------------------
+// ICONS
+//--------------------
+banana
+drowning
+    made by dalas
+    I release these under CC BY 4.0
+
+berserker
+brains
+cage
+coins
+escape-hatch
+ninja
+ogre
+sovereign
+    made by dalas by modifying pre-existing wesnoth images
+    I release these under CC BY 4.0
+
+clock (keith333) - CC BY 3.0
+    https://opengameart.org/content/clocks-watches-seamless-texture-with-normalmap-2
+    modified by dalas; I release my modified image under CC BY 4.0
+
+fiddle (IgnasD) - CC BY SA 3.0
+    https://opengameart.org/content/violin-0
+    modified by dalas; I release my modified image under CC BY 4.0 (or CC BY SA 3.0 if that's required by the original license)
+
+planb (Q_x) - GPL 3.0
+    https://opengameart.org/content/dwarven-map
+    modified by dalas; I release my modified image under CC BY 4.0
```

Comment:
GPL can't be relicensed to CC BY, for any version of those licenses.

More subtly, I think "GPL 3.0 only" isn't [allowed in mainline](https://wiki.wesnoth.org/Wesnoth:Copyrights), but need to clarify that. As written, "GPL v2 or later" is allowed, but I think that's "offers the recipient the right to redistribute it under GPL v2 or, at the recipient's choice, any later version"; so it needs to be a superset of "GPL v3 or later".